contentcuration/contentcuration/frontend/channelEdit/components/RelatedResourcesList/RelatedResourcesList.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+
+  <VList
+    v-if="items && items.length"
+    two-line
+  >
+    <template v-for="(item, itemIdx) in items">
+      <VListTile
+        :key="`list-tile-${itemIdx}`"
+        data-test="resource"
+        @click="onItemClick(item.id)"
+      >
+        <VListTileAction>
+          <ContentNodeIcon
+            v-if="item.kind"
+            :kind="item.kind"
+            :size="20"
+          />
+        </VListTileAction>
+
+        <VListTileContent>
+          <VListTileTitle
```

Comment:
`notranslate` class 

There is also an [ActionLink component](https://github.com/learningequality/studio/blob/vue-refactor/contentcuration/contentcuration/static/js/edit_channel/sharedComponents/ActionLink.vue) you can use that will take care of this styling for you if you'd like to use it